e2e/src/editCra.e2e-spec.ts

```diff
@@ -0,0 +1,187 @@
+      workingDays += (weekDay === 0 || weekDay === 6) ? 0 : 1;
+    }
+
+    return workingDays;
+  };
+
+  beforeEach(() => {
+    editCraPage = new EditCraPage();
+    editCraPage.navigateTo();
+    browser.waitForAngular();
+
+    nameInput = editCraPage.getInputByLabelText('Nom');
+    emailInput = editCraPage.getInputByLabelText('Email');
+    titleInput = editCraPage.getInputByLabelText('Intitulé');
+    clientInput = editCraPage.getInputByLabelText('Client final');
+
+    selectedNumber = editCraPage.getElementsByText('Nombre total de journées :')
+            .first()
+            .element(by.xpath('..//span[@class="highlight"]'));
+  });
+
+  it('should be possible to enter my name', () => {
+    nameInput.sendKeys('Tester');
+    expect(nameInput.getAttribute('value')).toBe('Tester');
+  });
+
+  it('should be possible to enter my email', () => {
+    emailInput.sendKeys('tester@test.com');
+    expect(emailInput.getAttribute('value')).toBe('tester@test.com');
+  });
+
+  it('should be possible to enter a mission title', () => {
+    titleInput.sendKeys('Testing');
+    expect(titleInput.getAttribute('value')).toBe('Testing');
+  });
+
+  it('should be possible to enter the client\'s name', () => {
+    clientInput.sendKeys('Test.com');
+    expect(clientInput.getAttribute('value')).toBe('Test.com');
+  });
+
+  describe('On the calendar', () => {
+    let aDayCell: ElementFinder;
+
+    beforeEach(() => {
+      aDayCell = editCraPage.getElementsByText('4').first();
+      aDayCell.click();
+    });
+
+    it('should be possible to select days', () => {
+      expect(editCraPage.getADayBadge(aDayCell).getText()).toBe('1\njour');
+    });
+
+    it('should be possible to select every working day', () => {
+      editCraPage.getElementsByText('Sélectionner les jours ouvrés').first().click();
+      expect(selectedNumber.getText()).toBe(getBusinessDaysNumber().toString());
+    });
+
+    describe('', () => {
+      let dayBadge: ElementFinder;
+
+      beforeEach(() => {
+        dayBadge = editCraPage.getADayBadge(aDayCell);
+
+        browser.actions()
+            .mouseMove(dayBadge)
+            .perform();
+
+        // Sleeping to wait for the annimation end
+        browser.sleep(50);
+
+        browser.actions()
+            .click(editCraPage.getElementsByText('0,5').first())
+            .perform();
+
+        // Sleeping to wait for the annimation end
+        browser.sleep(50);
+
+        // Moving the mouse away to read the value
+        browser.actions()
+            .mouseMove(aDayCell)
+            .perform();
+
+        // Sleeping to wait for the annimation end
+        browser.sleep(170);
+      });
+
+      it('should be possible to change the duration of a day', () => {
+        expect(dayBadge.getText()).toBe('0,5\njour');
+      });
+
+      it('should show the total number of days', () => {
+        editCraPage.getElementsByText('10').first().click();
+        expect(selectedNumber.getText()).toBe('1,5');
```

Comment:
Why you click on another day ? You already test this case.

Test if the selectedNumber is `0.5` is enough no ?